tests/e2e/e2e_egress_validator_test.go

```diff
@@ -1,12 +1,38 @@
 package e2e
 
 import (
+	"context"
+
+	"github.com/google/uuid"
+
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	pol "github.com/openservicemesh/osm/pkg/apis/policy/v1alpha1"
 	. "github.com/openservicemesh/osm/tests/framework"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var _ = OSMDescribe("Submit Egress Policy",
-	OSMDescribeInfo{},
+	OSMDescribeInfo{
+		Tier:   2,
+		Bucket: 5,
+	},
 	func() {
-		It("Tests a valid Egress Policy", func() {})
+		Context("EgressValidator", func() {
+			It("Tests a valid Egress Policy", func() {
+				ctx := context.TODO()
+				egressin := new(pol.Egress)
+				egressin.SetName(uuid.New().String())
```

Comment:
nit: there's a uuid.NewString() func

Reply:
cool. i don't think in real life this will last unless we want people to be able to test ad nauseum. there is actually default validation logic that blocks dups..since this is not intended to be run interactively that might be ok to bypass but if someone goes nuts that's a lot of policies!